pkgs/applications/graphics/gimp/wrapper.nix

```
@@ -13,8 +13,14 @@ let
     lib.attrValues gimpPlugins
   );
   selectedPlugins = lib.filter (pkg: pkg != gimp) (if plugins == null then allPlugins else plugins);
-  extraArgs = map (x: x.wrapArgs or "") selectedPlugins;
-  versionBranch = lib.versions.majorMinor gimp.version;
+  extraArgs =
+    map (x: x.wrapArgs or "") selectedPlugins
+    ++ lib.optionals (gimp.majorVersion == "2.0") [
+      ''--prefix GTK_PATH : "${gnome-themes-extra}/lib/gtk-2.0"''
+    ];
+  exeVersion =
+    if gimp.majorVersion == "2.0" then lib.versions.majorMinor gimp.version else gimp.majorVersion;
+  majorVersion = if gimp.majorVersion == "2.0" then "2" else "3";
```

```suggestion
  majorVersion = lib.versions.major gimp.version;
```
we can now also probably simplify the above line